data/examples/hf_compare/compare.py

```diff
@@ -32,25 +32,25 @@ def main(fns_fchk):
         g09_energy = mol.energy
 
         # Compute Gaussian integrals.
-        olp = mol.obasis.compute_overlap(mol.lf)
-        kin = mol.obasis.compute_kinetic(mol.lf)
-        na = mol.obasis.compute_nuclear_attraction(mol.coordinates, mol.pseudo_numbers, mol.lf)
-        er = mol.obasis.compute_electron_repulsion(mol.lf)
+        olp = mol.obasis.compute_overlap()
+        kin = mol.obasis.compute_kinetic()
+        na = mol.obasis.compute_nuclear_attraction(mol.coordinates, mol.pseudo_numbers)
+        er = mol.obasis.compute_electron_repulsion()
 
         # Make a list of the expansion objects.
-        exps = [mol.exp_alpha]
-        if hasattr(mol, 'exp_beta'):
-            exps.append(mol.exp_beta)
+        orbs = [mol.orb_alpha]
+        if hasattr(mol, 'orb_beta'):
+            orbs.append(mol.orb_beta)
 
         # Keep the g09 dms.
         if debug:
-            dms_g09 = [exp.to_dm() for exp in exps]
+            dms_g09 = [orb.to_dm() for orb in orbs]
 
         # Construct an initial guess.
-        guess_core_hamiltonian(olp, kin, na, *exps)
+        guess_core_hamiltonian(olp, kin + na, *orbs)
```

Comment:
My intention was to simplify the API and also to have less type checking in the functions. If you look at the argument list of the old version and the documentation, it took too much Python skill to understand how it works.